orchestrator/settings.py

```diff
@@ -72,11 +72,11 @@ class Oauth2Settings(BaseSettings):
 oauth2_settings = Oauth2Settings()
 
 # Tracer settings
-jaeger_exporter = jaeger.JaegerSpanExporter(
-    service_name=app_settings.SERVICE_NAME,
+jaeger_exporter = JaegerExporter(
+    # service_name=app_settings.SERVICE_NAME,
     agent_host_name=app_settings.LOGGING_HOST,
-    insecure=True,
+    # insecure=True,
```

Comment:
Why are these 2 lines commented out?

Reply:
The new JaegerExporter doesn't use them. Removed it.

Reply:
I seem to use the wrong Jaeger Exporter: the traces appear in the jaeger GUI with "service_name" => unknow.
Added a todo to keep track of it.